R/uvhydrograph-render.R

```diff
@@ -100,32 +115,31 @@ getSecondaryReportElements <- function(reportObject, month, timezone, excludeZer
   hasReferenceSeries <- hasReferenceSeries(reportObject)
   hasUpchainSeries <- hasUpchainSeries(reportObject)
   
-  if((hasReferenceSeries && !isPrimaryDischarge(reportObject)) || hasUpchainSeries) {
-    if(hasReferenceSeries) {
-      corrections <- parseCorrectionsByMonth(reportObject, "referenceSeriesCorrections", month)
-    } else {
-      corrections <- parseCorrectionsByMonth(reportObject, "upchainSeriesCorrections", month)
-    }
-    secondarySeriesList <- parseSecondarySeriesList(reportObject, month, timezone)
-    if(!isEmptyOrBlank(secondarySeriesList[['corrected']]) && !isEmptyVar(secondarySeriesList[['corrected']][['points']])){ #if corrected data exists
-      secondaryLims <- calculateLims(secondarySeriesList[['corrected']][['points']])
-      secondaryPlot <- createSecondaryPlot( 
-          readSecondaryTimeSeriesUvInfo(reportObject),
-          parseUvTimeInformationFromLims(secondaryLims, timezone),
-          secondarySeriesList, 
-          readSecondaryUvHydroApprovalBars(reportObject, timezone), 
-          readEffectiveShifts(reportObject, timezone, month),
-          readUvMeasurementShifts(reportObject, month),
-          readUvGageHeight(reportObject, month),
-          corrections, 
-          secondaryLims,
-          timezone, 
-          excludeZeroNegativeFlag, 
-          tertiary_label=getTimeSeriesLabel(reportObject, "effectiveShifts"))
-      secondaryTable <- parseCorrectionsAsTable(corrections)
-    } else {
-      secondary_status_msg <- paste('Corrected data missing for', fetchReportMetadataField(reportObject, 'secondaryParameter'))
-    }
+  if(hasReferenceSeries) {
```

Comment:
Those whole section used to be nested in an if block that toggled secondary plotting on/off. It's indented back because that logic has moved up a level. No functional changes here.